src/games/erdos_renyi.c

```diff
@@ -244,6 +244,49 @@ igraph_error_t igraph_erdos_renyi_game_gnm(
     return IGRAPH_SUCCESS;
 }
 
+igraph_error_t igraph_erdos_renyi_game_gnm_multi(
+    igraph_t *graph, igraph_integer_t n, igraph_integer_t m,
+    igraph_bool_t directed, igraph_bool_t loops
+) {
+
+    igraph_integer_t no_of_nodes = n;
+    igraph_integer_t no_of_edges = m;
```

Comment:
This is a simple renaming of existing variables, so let's just get rid of the `no_of_nodes` and `no_of_edges` and keep using `n` and `m` throughout.  Currently both are used, somewhat inconsistently.